.github/workflows/ci.yml

```diff
@@ -75,3 +75,75 @@ jobs:
 
         -   name: Run pytest
             run: pytest -sv tests
+
+    integration:
+
+            runs-on: ubuntu-latest
+
+            strategy:
+                matrix:
+                    python-version: ['3.9']
```

Comment:
Maybe cover more cases:
- Python version: 3.9, 3.11
- aiida-core version here, e.g.,  2.3, 2.4, 2.5?

Reply:
I agree with python, but maybe aiida-core we just test one? Just because it would take too many resources? They would be 6 (x2) running workflows, which seems an overshooting (one can simply test this in a local tox environment).
If we find a fix that works for both aiida-core 2.5 and >=2.6, we can then integrate two aiida-core versions. What do you think?